esupy/processed_data_mgmt.py

```diff
@@ -88,7 +89,10 @@ def download_from_remote(file_meta, paths, **kwargs):
                                           + '/' + subdirectory)
                 file = folder + "/" + f
                 create_paths_if_missing(file)
-                log.info('%s saved to %s', f, folder)
+                log.info('%s downloaded from '
+                         'https://dmap-data-commons-ord.s3.amazonaws.com/'
+                         'index.html?prefix=flowsa/ and saved to %s',
+                         f, folder)
```

Comment:
this shouldn't be flowsa specific

Reply:
Thanks! Fixed it [ae2e1a0](https://github.com/USEPA/esupy/commit/ae2e1a018605ccb650280a0a1b9c79ed14c66bf6)